app/public/pos/payment/script.js

```diff
@@ -0,0 +1,69 @@
+const params = new URLSearchParams(window.location.search);
+const orderId = params.get('id');
+const ticketItems = document.getElementById("ticket-table");
+const subtotalElement = document.getElementById("subtotal");
+const discount = 1.00; // placeholder - need to hash out specifics for discounts
+const tax = 0.08; // placeholder - need to hash out specifics for tax caluclation
+const totalElement = document.getElementById("total");
+const amountDue = document.getElementById("amountDue");
+const cashInput = document.getElementById("cashInput");
+const buttons = document.querySelectorAll(".num-btn");
+const clearButton = document.getElementById("clear");
+const changeElement = document.getElementById("change");
+const payButton = document.getElementById("pay");
+
+buttons.forEach(button => {
+    button.addEventListener("click", function () {
+        let value = button.getAttribute("data-value");
+        let currentValue = parseFloat(cashInput.value.replace('$', ''));
+
+        // handling for '.'
+        if (value === '.' && currentValue.includes('.')) {
```

Comment:
getting an error in `currentValue.includes('.')`, probably because it's a floating point number

Reply:
Oof overlooked that, but I'll drop the conditional for now since the idea of splitting the cash input was talked about